CoreScriptsRoot/CoreScripts/MainBotChatScript2.lua

```diff
@@ -566,7 +537,7 @@ end
 
 function addDialog(dialog)
 	if dialog.Parent then
-		if dialog.Parent:IsA("BasePart") then
+		if dialog.Parent:IsA("BasePart") and dialog:IsDescendantOf(game.Workspace) then
```

Comment:
We don't want the dialog GUI to display when the dialog object is parented to another service such as ReplicatedStorage or Lighting.

See report here: http://devforum.roblox.com/t/is-dialog-still-supported/29359/7